src/_info.sh

```diff
@@ -26,13 +26,14 @@ function mod_info() {
   msg "    %s\n" "$active"
   echo >&2 ""
 
+  #Check for Maintenance Mode
+  msg check_for_maintenance_mode
```

Comment:
the logging cmd should happen in only one place, so I'd say drop the `msg` here and let the `info` in the function itself handle things.
```suggestion
  check_for_maintenance_mode
```